scripts/cat/cats.py

```diff
@@ -2079,7 +2079,7 @@ def is_potential_mate(self,
             # if only one is aged up at this point, later they are more moons apart than the setting defined
             # game_config boolian "override_same_age_group" disables the same-age group check.
             if game.config["mates"].get("override_same_age_group", False) or self.age != other_cat.age:
-                if abs(self.moons - other_cat.moons) > game.config["mates"]["age_range"] + 1:
+                if abs(self.moons - other_cat.moons) or (other_cat.moons - self.moons) > game.config["mates"]["age_range"] + 1:
```

Comment:
Checking the negative of the difference should not change the results, since we are checking the absolute value.  In addition, the logic of this is incorrect. As it is, this will always evaluate to True is the cat's ages are different.